tests/test_util.py

```diff
@@ -0,0 +1,15 @@
+from hyp3_gamma import util
+
+
```

Comment:
some of the other functions that were moved to `util` could deserve tests, but not a priority with this PR

Reply:
Agreed -- I think there will be a refactoring push after this.